lib/guardian/user_guardian.rb

```diff
@@ -158,4 +158,13 @@ def can_see_review_queue?
   def can_see_summary_stats?(target_user)
     true
   end
+
+  def can_upload_profile_header?(user)
+    (is_me?(user) && user.trust_level >= TrustLevel[SiteSetting.min_trust_level_to_allow_profile_background.to_i]) || is_staff?
```

Comment:
Check out `user.has_trust_level?` It will also check for staff so you don't have to.

Reply:
said changes done